sources/packages/web/src/services/AuthService.ts

```diff
@@ -267,6 +267,13 @@ export class AuthService {
         break;
       }
       case ClientIdType.Student: {
+        // Send the user to login page with invalid beta user message instead of dashboard.
+        this.priorityRedirect = {
+          name: StudentRoutesConst.INVALID_BETA_USER,
+        };
```

Comment:
To do the specific redirect to `INVALID_BETA_USER` should we move this inside the `if (options?.invalidBetaUser)`?

Reply:
yes i had the same question.

Reply:
Yes, you are right. Changed. Thanks!